agent/config/pbench-agent-default.cfg

```diff
@@ -10,30 +17,32 @@ dir = /srv/pbench/public_html/incoming
 scp_opts = -o BatchMode=yes -o StrictHostKeyChecking=no
 ssh_opts = -o BatchMode=yes -o StrictHostKeyChecking=no
 
-[pbench-agent]
-install-dir = %(pbench_install_dir)s
-pbench_user = pbench
-pbench_group = pbench
-pbench_run = /var/lib/pbench-agent
-pbench_log = %(pbench_run)s/pbench.log
-
```

Comment:
Was it important to move this section?

Reply:
This is back-port of the changes from PR #1719, as stated in the commit message and in the PR description.

We can debate the need to move this section, but I'd rather be faithful to the back-port.